boltdesignsystem/bolt
# Update the z-index for tooltip and popover

## Jira

https://pegadigitalit.atlassian.net/browse/DS-479

## Summary

Update default `z-index` to match earlier default for tooltip and popover.

## Details

`zIndex` prop was added to tippy call.

## How to test

Open popover and tooltip demo and confirm that element created by tippy on the bottom of the page use:
- `z-index: 120` for popovers

![image](https://user-images.githubusercontent.com/16049049/118635859-5c095f80-b7d4-11eb-9b16-0c7c56d3add6.png)
- `z-index: 140` for tooltips

![image](https://user-images.githubusercontent.com/16049049/118635736-37ad8300-b7d4-11eb-8126-5b148b12f07c.png)
